src/pages/index.js

```diff
@@ -1,12 +1,27 @@
 import * as React from "react";
 import { DefaultLayout } from "../components/DefaultLayout";
+import { Button } from "../components/Button";
+import { Container } from "../components/Container";
 
 const IndexPage = () => {
   return (
-    <main className="container m-auto">
+    <Container>
       <title>Portfolio page</title>
-      <DefaultLayout></DefaultLayout>
-    </main>
+      <DefaultLayout>
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-x-32 my-56">
+          <div>left content</div>
+          <div className="grid">
+            <p className="text-3xl mb-5">Hi! Nice to meet you!</p>
+            <p className="text-violet-light mb-8">
+              My name is <span className="text-white">Nadiia</span> and I'm a
+              digital artist. Here I will have a short introductory text and
+              perhaps a few other things. Exciting!
+            </p>
+            <Button>Browse my work</Button>
```

Comment:
I think this will technically be a `Link` that looks like a button (designers, am I right?).

Reply:
(the best one!)
yes. you are right this is the link. 
But it depends on what this link will do. Now we are going to have just one page. And I would ask where this line would lead us?

Reply:
I think to `#work`, the same as in the header. What do you think?